odoo/tests/common.py

```diff
@@ -657,6 +657,15 @@ def assertXMLEqual(self, original, expected):
     def assertHTMLEqual(self, original, expected):
         return self._assertXMLEqual(original, expected, 'html')
 
+    def profile(self, **kwargs):
+        test_method = getattr(self, '_testMethodName', 'Unknown test method')
```

Comment:
This is not documented in `unittest`, so it *might* change in the future.  This is safer:
```suggestion
        test_method = str(self)
```

Reply:
It is what we are using in OdooTestResult and TagsSelector. As far as I know str(self) would have a longer form.
Example:
test_assign_perf_duplicates
test_assign_perf_duplicates (odoo.addons.crm.tests.test_performances.TestLeadAssignPerf)

It feels a little long, especially if we want to have readable user/warmup state. 
What do you think?

Reply:
If we already use it, then it's okay, I guess.